### ISSUE TITLE
Non-standard extension should be behind flag

### ISSUE THREAD

**Issue by drott** [googlefonts/fontations#1221]
Compare https://github.com/googlefonts/fontations/pull/927

I would prefer if we kept not yet fully standardized extensions behind a flag to avoid breakage and being able to follow Chromium's release process.

**Comment by dfrg** [comment]
We can either feature gate this or just mask out the cubic bit on read with a comment that it should be re-enabled when the spec finalizes. I lean toward the latter unless we have a case where we really need to parse cubic glyf outlines. 

@rsheeter any preference?

**Comment by drott** [comment]
>  I lean toward the latter [...]

Yes, let's move forward with that, could you do a PR? Thanks in advance.
